Sources/Auth/World/AuthWorld.swift

```diff
@@ -0,0 +1,5 @@
+/// - Attention: If logging is not desired, use `.noop` instead of `.live`.
+/// ```
+/// var AuthWorld = AuthWorldClient.noop
+/// ```
+var AuthWorld = AuthWorldClient.live // swiftlint:disable:this identifier_name
```

Comment:
From different comment, I am not using this anymore. Again, this was one approach in many, I only used it because we already use this approach in player, but if we prefer to have it differently between modules, we can of course.
With the setup, client can not opt out. I made the change where the decision is moved to client through injecting a code block, you can see it [here](https://github.com/tidal-music/tidal-sdk-ios/pull/81/commits/e2c56167d671610733ba2d294e02d6a3e5213d7c).